pkg/cmd/pulumi/history.go

```diff
@@ -41,10 +32,11 @@ func newHistoryCmd() *cobra.Command {
 		Use:        "history",
 		Aliases:    []string{"hist"},
 		SuggestFor: []string{"updates"},
-		Short:      "[PREVIEW] Update history for a stack",
-		Long: `Update history for a stack
-
-This command lists data about previous updates for a stack.`,
+		Short:      "[DEPRECATED] Display history for a stack",
```

Comment:
Should we add `Hidden: true` so the command doesn't show up in `--help`?

Reply:
good idea :)